autofaiss/external/quantize.py

```diff
@@ -49,6 +49,7 @@ def build_index(
     metric_type: str = "ip",
     nb_cores: Optional[int] = None,
     make_direct_map: bool = False,
+    should_be_memory_mappable: bool = False,
```

Comment:
can you add this to the doc ?

Reply:
Ah yes, I just modified the readme

Reply:
can you add the parameter in the list of parameter in the readme ?

Reply:
Done! The doc might be a bit too much duplicated^^